docs/differences-from-v1.md

```diff
@@ -47,22 +86,123 @@ Analysis strategies in CLI 2.x have substantive improvements in correctness and
 ### Improved debug logging
 
 CLI 2.x has much better debug logging, including a new feature called "replay logging" which allows developers to perfectly reproduce a bug report given a replay log. This is made possible by stronger compile-time guarantees that ensure all effects that occur during analysis are logged for replay.
+### "Modules" are now "analysis targets"
+
+In 1.x, the CLI was configured by "modules", which were entrypoints to user programs that the CLI should analyze. In 2.x, we've renamed "modules" to "analysis targets", and simplified how analysis targets are defined. We renamed analysis targets to avoid confusing naming conflicts with existing language-specific "modules". Analysis targets are no longer defined by multiple, confusingly similar fields (e.g. "Dir" vs. "Path"), and are now simple `(type, target)` tuples.
+
+## Breaking changes in FOSSA 2.x
+
+### Build Managers
```

Comment:
You should link the fossa deps doc here and write a blurb about how writing a custom integration is better for these build tools.